incubator/binding-mqtt-kafka/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaPublishProxyIT.java

```diff
@@ -110,6 +110,46 @@ public void shouldAbortWhenServerSentData() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("proxy.yaml")
+    @Specification({
+        "${mqtt}/publish.retained.server.sent.abort/client",
+        "${kafka}/publish.retained.server.sent.abort/server"})
+    public void shouldReceiveServerSentRetainedAbort() throws Exception
```

Comment:
Please review these test method names vs script scenario names.
For example, this one seems like it should be `shouldPublishRetainedThenReceiveServerSentAbort`